Missing COVERALLS_REPO_TOKEN in error message

## Version

- php: 7.4
- php-coveralls: 2.4.1

## Setup

Run GitHub Workflow with env `COVERALLS_REPO_TOKEN`.

```php
name: CI

on:
  - push
  - pull_request

jobs:
  run-tests:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Setting up PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: 7.4
          extensions: curl, json, mbstring
          coverage: xdebug
          tools: composer

      - name: Install dependencies with composer
        run: composer install  --no-interaction --no-progress --no-suggest --prefer-dist

      - name: Run PHPUnit
        run: php vendor/bin/phpunit --verbose --stderr --coverage-text --coverage-clover=build/logs/clover.xml

      # https://github.com/php-coveralls/php-coveralls#github-actions
      - name: Upload coverage results to Coveralls
        run: |
          composer global require php-coveralls/php-coveralls
          php-coveralls --coverage_clover=build/logs/clover.xml -vvv
```

## Expect

```
For Githib Actions users:
  - GITHUB_REF
  - GITHUB_ACTIONS
  - GITHUB_RUN_ID
  - GITHUB_EVENT_NAME
  - COVERALLS_REPO_TOKEN
```

## Actual

```shell
Load coverage clover log:
  - /home/runner/work/laravel-dummy/laravel-dummy/build/logs/clover.xml
Found 3 source files:
  -   0.00% src/Dummy.php
  -   0.00% src/Facades/Dummy.php
  -  82.35% src/ServiceProvider.php
Coverage:  73.68% (14/19)
Collect git info
Read environment variables
Requirements are not satisfied.
  - GITHUB_ACTIONS='true'
  - GITHUB_REF='refs/heads/master'
  - CI_NAME='github'
  - CI_JOB_ID='292634117'
  - CI_BRANCH='master'

Set environment variables properly like the following.
For Travis users:

  - TRAVIS
  - TRAVIS_BUILD_NUMBER
  - TRAVIS_JOB_ID

For CircleCI users:

  - CIRCLECI
  - CIRCLE_BUILD_NUM
  - COVERALLS_REPO_TOKEN

For Jenkins users:

  - JENKINS_URL
  - BUILD_NUMBER
  - COVERALLS_REPO_TOKEN

For AppVeyor users:

  - APPVEYOR
  - APPVEYOR_BUILD_NUMBER

For Githib Actions users:
  - GITHUB_REF
  - GITHUB_ACTIONS
  - GITHUB_RUN_ID
  - GITHUB_EVENT_NAME

From local environment:

  - COVERALLS_RUN_LOCALLY
  - COVERALLS_REPO_TOKEN

elapsed time: 0.026 sec memory: 4.00 MB
Error: Process completed with exit code 1.
```

--- a/src/Bundle/CoverallsBundle/Entity/Exception/RequirementsNotSatisfiedException.php
+++ b/src/Bundle/CoverallsBundle/Entity/Exception/RequirementsNotSatisfiedException.php
@@ -78,6 +78,7 @@ For Github Actions users:
   - GITHUB_ACTIONS
   - GITHUB_RUN_ID
   - GITHUB_EVENT_NAME
+  - COVERALLS_REPO_TOKEN
 
 From local environment:
 

